src/api/protectedApiClient.ts

```diff
@@ -6,6 +6,25 @@ import { EventAnnouncement } from '../containers/singleEvent/ducks/types';
 import { ContactInfo } from '../containers/setContacts/ducks/types';
 import { EventInformation } from '../containers/upcoming-events/ducks/types';
 
+export interface User {
+  id: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  phoneNumber: string;
+}
+
+export interface PFRequest {
+  id: string;
+  user: User;
+}
+
+export interface PFRequestResponse {
+  data: {
```

Comment:
❓ Is this data from Axios? I think for all of our API requests we grab res.data, so this interface should just have requests (I'll point it out in the code below)